.changeset/violet-brooms-warn.md

```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/admin.login-flow.ai.v1": minor
```

Comment:
In this PR, we are not adding any new features to both these modules, but enabling a feature for the console application. Therefore, console should be updated to a minor version and both these modules should be updated to a patch version, IMO. 

WDYT @brionmario ?

Reply:
`patch` is meant for bug fixes. So, all these packages should get a `minor` bump.

Reply:
This is fixed with https://github.com/wso2/identity-apps/pull/7391/commits/501827bfce53bd4964d2c10dd035d056e556aa55